workspace/viewer/PdfJsViewer.ts

```diff
@@ -310,6 +310,7 @@ export class PdfJsViewer {
               private readonly logger: Logger,
               private readonly utils: Utils,
               private readonly i18n: I18N) {
+    this.pdfPageView = [];
```

Comment:
Please change the type and the name of this member variable, because it is not a single PDFPageView anymore.

Reply:
I changed the name pdfPageView to pdfPagesView, but didn't figure out yet how to create an array of PDFPageView.PDFPageView objects, because it throws TS2503 error "Cannot find namespace PDFPageView".

Reply:
Perhaps TypeScript type definitions in `node_modules` diverged from the real js code (we have outdated pdfjs library in this repository).
Anyway, the edit distance between `pdfPageView` and `pdfPagesView` is next to nothing while conceptually these objects are "two big differences", and array  is not a view at all. Rename it to `allPages` or `loadedPages` or anything else which matches the semantics.